module2/homework/transformContainers/test.cpp

```diff
@@ -1,6 +1,11 @@
-#include "gtest/gtest.h"
+#include <algorithm>
+#include <deque>
+#include <list>
+#include <map>
+#include <string>
```

Comment:
"uniquemap.hpp" has deque, list,map and string so you do not need to include them here.